### ISSUE TITLE
How to construct `fpField` in v0.39 ?

### ISSUE THREAD

**Issue by sumiya11** [Nemocas/Nemo.jl#1614]
Hi,

The constructor `GF(::Int) --> ::fpField` seems to be gone in v0.39, is this indended? Thanks

**Comment by thofma** [comment]
Yes, this is intended. There is now only finite field type. You can get an `fpField` by calling `Native.GF`.

**Comment by sumiya11** [comment]
Could you please share some insight into the reason for such change?
My previous understanding was that `fpFieldElem` is considerably faster than `FqFieldElem`, at least for basic arithmetic, and thus it could be a better as a default

**Comment by thofma** [comment]
Yes, for basic arithmetic, `fpFieldElem` is faster. But we noticed over the years, that having four different types for finite fields can be quite confusing for ordinary users. So we make the better user experience the default and keep the 'old' finite fields around.

**Comment by pogudingleb** [comment]
@thofma  Thank you for all the hard work for the unification! I would like to follow-up on @sumiya11 question: how reasonable would be to still use `fpFieldElem` if the efficiency is important? Will it be maintained further or it is now on the road to become depreciated?

**Comment by thofma** [comment]
There is no plan to deprecate it. We also use it internally downstream in our other packages. The code is pretty much stable, so the maintenance cost is very low.

**Comment by pogudingleb** [comment]
Great, thanks for clarifying!
